### ISSUE TITLE
Searched Image // Back Button Behaviour wrong 

### ISSUE THREAD

**Issue by SerenaButler** [LycheeOrg/Lychee-Laravel#90]
When clicking an Image within a search result, the back button at the image page should point (back) to the searched image results, **NOT** the Album of the image.

**Comment by ildyria** [comment]
That one is tricky to solve.

**Comment by SerenaButler** [comment]
Yes, back-Butttons with different "backs" are annoying sometimes ...
